apps/expo/package.json

```diff
@@ -16,30 +16,30 @@
   },
   "dependencies": {
     "@bacons/text-decoder": "^0.0.0",
-    "@expo/metro-config": "^0.18.11",
+    "@expo/metro-config": "^0.19.4",
     "@shopify/flash-list": "1.7.1",
     "@tanstack/react-query": "catalog:",
     "@trpc/client": "catalog:",
     "@trpc/react-query": "catalog:",
     "@trpc/server": "catalog:",
-    "expo": "~51.0.38",
-    "expo-constants": "~16.0.2",
-    "expo-dev-client": "~4.0.28",
-    "expo-linking": "~6.3.1",
-    "expo-router": "~3.5.23",
-    "expo-secure-store": "^13.0.2",
-    "expo-splash-screen": "~0.27.6",
-    "expo-status-bar": "~1.12.1",
-    "expo-web-browser": "^13.0.3",
+    "expo": "~52.0.7",
+    "expo-constants": "~17.0.3",
+    "expo-dev-client": "~5.0.2",
```

Comment:
@juliusmarminge, Did you consider dropping the `expo-dev-client` in favor of Expo Go? It could speed up the process for new people to take off and remove the initial friction.

Reply:
> Did you consider dropping the expo-dev-client in favor of Expo Go? It could speed up the process for new people to take off and remove the initial friction.

You need dev builds to run packages which uses native bindings. https://docs.expo.dev/develop/development-builds/introduction/

Reply:
The template lacks native libraries, so Expo Go can work initially. I've only considered the initial bootstrap simplicity; this is one line to document how to add Dev Builds.

Reply:
Yea but I think almost any app will eventually have native libraries and so setting you up for success is preferred. Otherwise people will start submitting issues why something isn't working

Reply:
I agree that when apps progress, there is a high probability the dev build is needed, but it is unlikely that at the beginning.

I'm resolving the comment, and thank you for the discussion!